ui/tests/acceptance/dashboard-test.js

```diff
@@ -16,27 +16,21 @@ import {
 } from '@ember/test-helpers';
 import { setupApplicationTest } from 'vault/tests/helpers';
 import { setupMirage } from 'ember-cli-mirage/test-support';
-import { create } from 'ember-cli-page-object';
 import { selectChoose } from 'ember-power-select/test-support/helpers';
 import { runCommands } from 'vault/tests/helpers/pki/pki-run-commands';
```

Comment:
You should be able to remove all `runCommands` and replace with `runCmd`. This will clean this up a bit because these are imported from pki and ideally we'd only want them used for pki specific tests.

Reply:
I've been in this deep for ember data, so for some additional context in PKI we're moving towards runCmd when possible. 

Below is the pki runCmd:
```
export const runCommands = async function (commands) {
  try {
    await consoleComponent.runCommands(commands);
    const res = consoleComponent.lastLogOutput;
    if (res.includes('Error')) {
      throw new Error(res);
    }
    return res;
  } catch (error) {
    // eslint-disable-next-line no-console
    console.error(
      `The following occurred when trying to run the command(s):\n ${commands.join('\n')} \n\n ${
        consoleComponent.lastLogOutput
      }`
    );
    throw error;
  }
};
```

Reply:
Vs the vault tools runCmd that should be the one we lean on first.
```
export async function runCmd(commands, throwErrors = true) {
  if (!commands) {
    throw new Error('runCmd requires commands array passed in');
  }
  if (!Array.isArray(commands)) {
    commands = [commands];
  }
  await cc.runCommands(commands);
  const lastOutput = cc.lastLogOutput;
  if (throwErrors && lastOutput.includes('Error')) {
    throw new Error(`Error occurred while running commands: "${commands.join('; ')}" - ${lastOutput}`);
  }
  return lastOutput;
}
```

they are more or less the same.

Reply:
On the [ember data branch](https://github.com/hashicorp/vault/pull/24760/files#diff-bdeca88952ac61fa3fabaa96667aff9c4dd796125ce9681c270beca64e4b4d79) we have a note to replace all uses of the pki runCommand with the vault tools one (which we update a bit in that project).